float32 detection_id
geometry_msgs/Point position